src/main/java/org/broadinstitute/hellbender/tools/recalibration/RecalDatum.java

```diff
@@ -33,6 +34,8 @@
 
     /**
      * number of bases seen that didn't match the reference
+     * (actually sum of the error weights - so not necessarily a whole number)
+     * Stored with an internal multiplier to keep it closer to the floating-point sweet spot and avoid numerical error (see TestMath.java for demonstration).
```

Comment:
Maybe reference this issue on github instead of TestMath?  It just seems strange to have a bunch of tests checked in that don't actually test anything.